selftests/functional/test_plugin_jobscripts.py

```diff
@@ -114,7 +150,7 @@ def test_non_existing_dir(self):
         config = script.TemporaryScript("non_existing_dir.conf",
                                         SCRIPT_NON_EXISTING_DIR_CFG % self.pre_dir)
         with config:
-            cmd = './scripts/avocado --config %s run passtest.py' % config
+            cmd = './scripts/avocado --config %s run /bin/true' % config
```

Comment:
@dmonakhov this is a collection of my suggestions for this commit: https://github.com/clebergnu/avocado/commit/7885b2380db7f209d9b878571a1f83269c986fb1

Also, please add a "Signed-off-by" line to your commit.